en/How to/Use callouts.md

```diff
@@ -206,5 +206,5 @@ We choose the syntax because:
 
 - it falls back to blockquotes in unsupported markdown renderers
 - it parses all Markdown, including links, embeds, etc.
-- it recognizes [[Internal link|Internal link's]] and will shows up in [[Backlinks]], [[Outgoing links]], and [[Graph view]]
+- it recognizes [[Internal link|Internal links]], will show up in [[Backlinks]], [[Outgoing links]], and [[Graph view]]
```

Comment:
```suggestion
- it recognizes [[Internal link|Internal links]], and shows up in [[Backlinks]], [[Outgoing links]], and [[Graph view]]
```

Thanks for spotting it! A couple of comments:

- Use active tense (prefer "shows" over "will show").
- "and" is replaced with a serial comma here, which turns two lists of different things into one, which I don't think is the intention. Adding the "and" back in there.